flume-ng-core/src/main/java/org/apache/flume/serialization/EventDeserializerType.java

```diff
@@ -26,6 +26,7 @@
 public enum EventDeserializerType {
   LINE(LineDeserializer.Builder.class),
   AVRO(AvroEventDeserializer.Builder.class),
+  FLUME(FlumeEventAvroEventDeserializer.Builder.class),
```

Comment:
I think `FLUME_AVRO` would be a more expressive name for this.
Please add this to the user guide as well. Thanks.